docs/docs/shards/index.md

```diff
@@ -164,4 +164,41 @@ Showing how we mix the Clojure-like language as a template language
     [info] [2021-04-04 21:05:50.123] [T-285088] [logging.cpp::53] [counter-10] 20
     ```
 
+## Shards' types
```

Comment:
this sounds like types of shards.
Should be more like types of values shards can manipulate (in short) kind of thing